### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by HarleyTwain** [aubingazhib/LightGlueStick#2]
Great work! I found that the loss in the paper might be a combination of LightGlue and GlueStick. But to avoid errors during the reproduction process, could you please provide a complete loss function? Thank you very much.

**Comment by aubingazhib** [comment]
Hi, I have a PR in glue-factory, where you could train LightGlueStick. https://github.com/cvg/glue-factory/pull/158

The loss functions etc. are defined there. But I may update the code a bit to make sure the results are fully reproducible.

**Comment by HarleyTwain** [comment]
I've found it. Thank you very much!
